src/modules/fw_autotune_attitude_control/fw_autotune_attitude_control.hpp

```diff
@@ -204,7 +222,15 @@ class FwAutotuneAttitudeControl : public ModuleBase<FwAutotuneAttitudeControl>,
 		(ParamFloat<px4::params::FW_YR_P>) _param_fw_yr_p,
 		(ParamFloat<px4::params::FW_YR_I>) _param_fw_yr_i,
 		(ParamFloat<px4::params::FW_YR_FF>) _param_fw_yr_ff,
-		(ParamFloat<px4::params::FW_Y_RMAX>) _param_fw_y_rmax
+		(ParamFloat<px4::params::FW_Y_RMAX>) _param_fw_y_rmax,
+
+		(ParamFloat<px4::params::FW_AT_SYSID_F0>) _param_fw_sysid_start_frequency,
```

Comment:
```suggestion
		(ParamFloat<px4::params::FW_AT_SYSID_F0>) _param_fw_at_sysid_f0,
```
it's not enforced automatically but using the same name makes the param search easier